addons/binding/org.openhab.binding.draytonwiser/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,429 @@
+        <label>Unified Signal Strength</label>
+        <description>The reported network signal strength</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="batteryLevel-channel" advanced="true">
+        <item-type>String</item-type>
+        <label>Battery Level</label>
+        <description>Current Battery Level for the device</description>
+        <state readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="unifiedBatteryLevel-channel">
+        <item-type>Number</item-type>
+        <label>Unified Battery Level</label>
+        <description>Current Battery Level for the device</description>
+        <state readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="heatChannel1Demand-channel">
+        <item-type>Number</item-type>
+        <label>Channel 1 Heat Demand</label>
+        <description>Current heat demand in %</description>
+        <state readOnly="true" pattern="%d %%"></state>
+    </channel-type>
+    
+    <channel-type id="heatChannel2Demand-channel">
+        <item-type>Number</item-type>
+        <label>Channel 2 Heat Demand</label>
+        <description>Current heat demand in %</description>
+        <state readOnly="true" pattern="%d %%"></state>
+    </channel-type>
+    
+    <channel-type id="heatChannel1DemandState-channel">
+        <item-type>Switch</item-type>
+        <label>Channel 1 Requesting Heat</label>
+        <description>Current channel is requesting heat</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="heatChannel2DemandState-channel">
+        <item-type>Switch</item-type>
+        <label>Channel 2 Requesting Heat</label>
+        <description>Current channel is requesting heat</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="hotWaterDemandState-channel">
+        <item-type>Switch</item-type>
+        <label>Hot Water Requesting Heat</label>
+        <description>Hot Water is requesting heat</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="awayModeState-channel">
+        <item-type>Switch</item-type>
+        <label>Away Mode Active</label>
+        <description>Away mode has been activated</description>
+    </channel-type>
+    
+    <channel-type id="awayModeSetPoint-channel" advanced="true">
+        <item-type>Number</item-type>
+        <label>Away Mode Set Point</label>
+        <description>Away Mode Set Point in degrees celsius</description>
+        <category>Temperature</category>
+        <state readOnly="false" pattern="%.1f °C"></state>
+    </channel-type> 
+    
+    <channel-type id="ecoModeState-channel">
+        <item-type>Switch</item-type>
+        <label>Eco Mode Enabled</label>
+        <description>Eco Mode activation state (EcoIQ)</description>
+    </channel-type>
+    
+    <channel-type id="manualModeState-channel">
```

Comment:
I am wondering whether eco and manual mode can both be set to "on"... if not, how about providing just one `mode` channel` with 2 options?

Reply:
Both can be on, in the state-model of the boiler controller at least. In manual mode I'm assuming it will still use it's eco-mode logic to maintain the manual setpoint.